designs/2021-init-command-eslint-cli/README.md

```diff
@@ -42,7 +42,7 @@ The approach would be following these steps
   - `json-stable-stringify-without-jsonify`
   - `cross-spawn`
   - `debug`
-  - `@eslint/eslintrc` (to be removed)
+  - `@eslint/eslintrc`
```

Comment:
I don’t think this is needed

Reply:
Hmmm, no good alternative I can think of. Please disregard. We can likely remove it when we switch to the new config system.